ictv/database.py

```diff
@@ -109,8 +109,14 @@ def update_database():
         DBVersion(version=database_version)
     db_version = DBVersion.select().getOne().version
     if db_version < 1:
-        # Do upgrade stuff
-        pass
+        print('Updating database to version %d' % 1)
+        table = User.sqlmeta.table
+        from datetime import datetime
+        column_sql = User.sqlmeta.getColumns()['creation_date'].sqliteCreateSQL()  + ' DEFAULT ' + datetime.now()
```

Comment:
I think the way SQLObject works internally is that it provides the default value at each row insertion, see my previous comment on the `default=` thing. It does not set a non-null constraint on the column, but always makes sure to provide a default value.

Reply:
You're probably right.